source/fx-guidelines/introduction.md

```diff
@@ -0,0 +1,23 @@
+# Introduction
+___
+
+The purpose of the Mattermost FX Guidelines doc is to align our team around a set of standards to reduce ambiguity in building our fan experience and file tickets to course correct when we deviate from the standards we set.
+
+## Quality Gates
+
+As our team and service evolve, the number and precision level of our standards will also increase, which in turn increases product quality.
+
+___
+
+## Principles
+
+As our team and service evolve, the number and precision level of our standards will also increase, which in turn increases product quality.
+
+### Responsive (Fast)
```

Comment:
Propose going down to heading level 4 (####) for these?It's what I've been doing for most docs because it reads better with less empty space